### PR TITLE
Feature/atr 262 master release notes15

### PR DESCRIPTION
**Pull request by k-s-popova**
Please review the release notes for 1.5 and the descriptions of the new diagnostics.

### CODE REVIEW

File: docs/diagnostics/PX1085.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# PX1085
+This document describes the PX1085 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                                     | Type                           | Code Fix    | 
+| ------ | ----------------------------------------------------------------------------------------------------- | ------------------------------ | ----------- | 
+| PX1085 | BQL statements and other database queries should not be executed during the `PXGraph` initialization. | Warning (Level 1: Significant) | Unavailable |
```
Comment (metallmonster) [inline docs/diagnostics/PX1085.md]: The diagnostic is executed only if ISV specific option is activated.

File: docs/diagnostics/PX1085.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# PX1085
+This document describes the PX1085 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                                     | Type                           | Code Fix    |
```
Comment (metallmonster) [inline docs/diagnostics/PX1085.md]: The type of the diagnostic is Error, not Warning

File: docs/ReleaseNotes.md [hunk 3]
```diff
@@ -1,6 +1,36 @@
...
+| [PX1080](diagnostics/PX1080.md) | Data view delegates should not start long-running operations.                                            | Error   | Available   | Unavailable |
+| [PX1081](diagnostics/PX1081.md) | Actions cannot be executed during the `PXGraph` initialization.                                          | Error   | Available   | Unavailable |
+| [PX1082](diagnostics/PX1082.md) | Actions cannot be executed within data view delegates.                                                   | Error   | Available   | Unavailable |
+| [PX1083](diagnostics/PX1083.md) | Changes cannot be saved to the database from data view delegates.                                        | Error   | Available   | Unavailable |
+| [PX1084](diagnostics/PX1084.md) | `PXGraph` instances cannot be initialized within data view delegates.                                    | Error   | Available   | Unavailable |
+| [PX1085](diagnostics/PX1085.md) | BQL statements and other database queries should not be executed during the `PXGraph` initialization.    | Warning | Available   | Unavailable |
```
Comment (metallmonster) [inline docs/ReleaseNotes.md]: The type of the diagnostic is Error, not Warning

File: docs/diagnostics/PX1085.md (hunk 1 above)
Comment (k-s-popova) [reply docs/diagnostics/PX1085.md]: This is written in the end of the description.

File: docs/diagnostics/PX1059.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+This document describes the PX1059 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                             | Type  | Code Fix    | 
+| ------ | ----------------------------------------------------------------------------- | ----- | ----------- | 
+| PX1059 | Changes to `PXCache` cannot be performed during the `PXGraph` initialization. | Error | Unavailable |
+
+## Diagnostic Description
+The `Insert`, `Update`, and `Delete` methods cannot be invoked on `PXCache` objects during the `PXGraph` initialization (that is, in `PXGraph` constructors, in the `Initialize` method overridden in `PXGraphExtension`, or in handlers subscribed at run time through the static `InstanceCreated` member of `PXGraph`). The invocation of these methods during the `PXGraph` initialization can lead to data inconsistency.
```
[the remark this answers is not in the document]
Comment (k-s-popova) [reply docs/diagnostics/PX1059.md]: fixed

File: docs/diagnostics/PX1085.md (hunk 2 above)
Comment (k-s-popova) [reply docs/diagnostics/PX1085.md]: @metallmonster , it is marked with ISVCertW1 in jira - that is, it was supposed to be a warning.

Comment (metallmonster) [reply docs/diagnostics/PX1085.md]: @k-s-popova I've changed severity of the diagnostic. The changes are in the following pull-request https://github.com/Acumatica/Acuminator/pull/158

**Comment by k-s-popova** [comment]
@metallmonster , I've added the description of the PX1086 diagnostic. Please review.

File: docs/diagnostics/PX1072.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+This document describes the PX1072 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                  | Type                           | Code Fix  | 
+| ------ | ---------------------------------------------------------------------------------- | ------------------------------ | --------- | 
+| PX1072 | BQL queries must be executed within the context of an existing `PXGraph` instance. | Warning (Level 1: Significant) | Available | 
+
+## Diagnostic Description
+The `new PXGraph()` statement should not be used in the execution of BQL queries. Instead, an existing `PXGraph` instance should be used. Because the creation of a `PXGraph` instance is an expensive operation, the instantiation of unnecessary graph instances slows the performance of the application. Also, when you create a new graph, you miss the query results that have already been cached in an existing graph.
```
Comment (dropsonic) [inline docs/diagnostics/PX1072.md]: `PXGraph.CreateInstance` should also be mentioned

File: docs/diagnostics/PX1073.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+## Diagnostic Description
+No exceptions of any type can be thrown in the `RowPersisted` event handlers. 
+
+The `RowPersisted` event handlers can be used for the following purposes:
+
+ - Retrieval of the data from the database
+ - Restoration of the values of the DAC fields if the status of the transaction scope is `Aborted`
+
+An exception thrown in a `RowPersisted` event handler prevents the saving of the data to the database but does not rall back changes in `PXCache` instances, which leads to data inconsistency.
```
Comment (dropsonic) [inline docs/diagnostics/PX1073.md]: r**A**ll back

File: docs/diagnostics/PX1072.md (hunk 5 above)
Comment (k-s-popova) [reply docs/diagnostics/PX1072.md]: fixed

File: docs/diagnostics/PX1073.md (hunk 6 above)
Comment (k-s-popova) [reply docs/diagnostics/PX1073.md]: fixed

**Pull request merged**